modules/asset_import/assimp.h

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2007-2017 Juan Linietsky, Ariel Manzur.
+// Copyright (c) 2014-2017 Godot Engine contributors (cf. AUTHORS.md)
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+// -- Godot Engine <https://godotengine.org>
+
+#ifndef EDITOR_SCENE_IMPORTER_ASSET_IMPORT_H
+#define EDITOR_SCENE_IMPORTER_ASSET_IMPORT_H
```

Comment:
The guard and the file name should match the name of the main class. So the file should be `editor_scene_importer_assimp.{cpp,h}`, and the guard `EDITOR_SCENE_IMPORTER_ASSIMP_H`.

Reply:
Resolved.